site/en/docs/extensions/reference/alarms/index.md

```diff
@@ -45,7 +45,9 @@ chrome.runtime.onInstalled.addListener(async ({ reason }) => {
 
 {% Aside %}
 
-Starting in Chrome 117, the number of active alarms is limited to 500. Once this limit is reached, `chrome.alarms.create()` will fail. When using a callback, [`chrome.runtime.lastError`][last-error] will be set. When using promises, the promise will be rejected.
+**Chrome 120:** Starting in Chrome 120, the minimum alarm interval has been reduced from 1min to 30s. Call    `await chrome.alarms.create('min-alarm', { periodInMinutes: 0.5 });` to set the alarm period to 30s.
```

Comment:
Not sure I like the way it renders, what do you think?
<img width="711" alt="Screenshot 2023-10-24 at 18 22 49" src="https://github.com/GoogleChrome/developer.chrome.com/assets/37001393/fa2c02fe-3fe8-44a6-8e89-dbe4591c3307">

Reply:
Which part don't you like? I liked it :-)

Reply:
The inline code felt a bit weird, but honestly it's just a personal preference. Otherwise LGTM 👍

Reply:
OK, I didn't like that part either ( I wanted to show the `0.5` though, because that was the part I was wondering about before). Shortened the code snippet. Wdyt?

Reply:
> OK, I didn't like that part either ( I wanted to show the `0.5` though, because that was the part I was wondering about before). Shortened the code snippet. Wdyt?

💯